ProjectTemplates/gmodnet-module-web/.gitignore

```diff
@@ -0,0 +1 @@
+TemplateModuleWeb.sln
```

Comment:
Why ignore solution file?

Reply:
In order to maintain compatibility with other .NET SDK templates, like `console`, `classlib`, etc.